docs/source/tutorials/AfriSAR_AGB_Maps_1681.ipynb

```diff
@@ -0,0 +1,1381 @@
+{
```

Comment:
<p>We should include references for this type of dataset documentation, since we didn't write it.</p><p> </p><p>I suggest adding the following line right after the <strong>Description</strong></p><p> </p><blockquote>Source: https://daac.ornl.gov/AFRISAR/guides/AfriSAR_AGB_Maps.html</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>